### DISCUSSION TITLE
Next labs firmware release for Hero 10 Black?

### DISCUSSION THREAD

**Discussion by aalse** [gopro/labs#156] [General]
Hi,

I have a Hero 10 Black and wanted to try out the GPS lock before starting recording feature which was discussed here https://github.com/gopro/labs/issues/82 

Is there any rough estimate on when (or if it will come at all) this funtionality will come to the Hero 10 Black labs firmware? I see a new Labs firmware for the Hero 11 is out as of November.

Regards,
Aditya

**Comment by dnewman-gpsw** [comment]
Maybe one more update for HERO10 can happen, I don't have the dates.  The Labs typically gets update when the main firmware has changes, the get less often as a camera matures.

**Reply by aalse** [reply]
As luck would have it, a new firmware (v1.50) for the Hero 10 Black released on Decembet 14 https://gopro.com/en/us/update/hero10-black . Hopefully this also translates into a Labs update, fingers crossed.

**Reply by dnewman-gpsw** [reply]
The year end holidays have slowed it public posting.  But it is done. 
Here it is https://gopro.github.io/labs/firmware/bin/LABS_HERO10_01_50_70.zip

**Reply by aalse** [reply]
Thank you, looking forward to trying it out!

**Reply by HuggyBearGP10** [reply]
I just stumbled upon this awesomeness. Thank you BTW..I'm curious, should I update to the recent regular update before updating the 1.50 labs file or does the labs contain that update plus the "labs" properties? Thanks in advance.

**Reply by dnewman-gpsw** [reply]
All firmware update are self contained and complete.  Just need to install the Labs version.
